allocations/templates/allocations/index.html

```diff
@@ -119,12 +119,12 @@ <h3>Allocations</h3>
                                     <th width="20%">Actions</th>
                                 </tr>
                             </thead>
-                            <tbody>
-                                <tr ng-repeat="allocation in project.allocations" ng-show="!allocation.doNotShow">
+                            <tbody ng-repeat="allocation in project.allocations" ng-show="!allocation.doNotShow">
```

Comment:
Huh, I didn't know it was legal in HTML to have multiple `tbody`s. But given the amount of things that are legal in HTML, nothing surprises me anymore.

Reply:
I didn't know I couldn't do that 👍 
But the internet says it's ok https://stackoverflow.com/a/3076721